data/ui_dashboards/demodata/Chart data.csv

```diff
@@ -0,0 +1,13 @@
+data point id,data point name,data point value,data point value label,data point series,data point primary category,data point secondary category,data point primary category label,data point secondary category label,data point time,data point time unit,data point color,data point description,data point order,included in chart
```

Comment:
I find the naming a bit confusing.
To many 'data point ...' and is this actually an 'EAV'?
But for now I don't mind that much.